osu.Game/Screens/Loader.cs

```diff
@@ -37,6 +39,11 @@ public Loader()
 
         private IntroScreen getIntroSequence()
         {
+            // Headless tests run too fast to load non-circles intros correctly.
+            // They will hit the "audio can't play" notification and cause random test failures.
+            if (SeasonalUIConfig.ENABLED && !DebugUtils.IsNUnitRunning)
```

Comment:
...except this fails the tests that expect the notification to pop up :see_no_evil: (`TestPlayIntroWithFailingAudioDevice`)

Reply:
I meant to remove this. Mentioned in OP.